pipe/psf_phot.py

```diff
@@ -124,6 +124,8 @@ def __init__(self, pipe_params):
                                 # Can be used as a dark current image
         self.im_thermFront_2 = None  # thermFront_2 vector, passed on to save-file
 
+        self.mad_sa = None
+        self.mad_im = None

```

Comment:
I've added these `mad_sa` and `mad_im` attributes so we can check the MADs later on in the tests.